### DISCUSSION TITLE
Ability to assign a Contact to an Interface

### DISCUSSION THREAD

**Discussion by dss70** [netbox-community/netbox#8056] [Ideas]
Usually a device interface is related to user, the user is connected to a switch interface. It would be useful if I could assing a user (a Contact) to a device interface. Right now we use the description field for this purpose.

**Comment by jeremystretch** [comment]
Honestly this seems like a fairly niche use case, and I don't think a direct relationship to the contact model would make sense. However, custom object fields (#7006) are planned for v3.2 and would probably suit your needs.
